plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/models/PortSpecAdapter.java

```diff
@@ -99,9 +99,15 @@ public boolean equals(Object obj) {
 		return true;
 	}
 	
-	
-	
-	private String getName(int port, String protocol) {
+    /**
+     * For better test reporting
+     */
+    @Override
+    public String toString() {
+        return "PortSpecAdapter [name=" + name + ", protocol=" + protocol + ", port=" + port + "]";
```

Comment:
using ToStringBuilder is probably not justified with such a small number of fields